server/opts.go

```diff
@@ -190,6 +190,8 @@ type Options struct {
 	WriteDeadline    time.Duration `json:"-"`
 	MaxClosedClients int           `json:"-"`
 	LameDuckDuration time.Duration `json:"-"`
+	// MaxTracedMsgLen is the maximum printable length for traced messages.
+	MaxTracedMsgLen int `json:"max_traced_msg_len"`
```

Comment:
We have deprecated use of `json` tags in Options because we used to use this directly in monitoring object but we no longer do.
As a side note, if this is something that we would want to see in `/varz` then it would have to be added in monitor.go. This can be done as separate PR though.

Reply:
Done. I have use `json:"-"` for `MaxTracedMsgLen`.